deploy/backplane/mtsre/ocs-consumer/01-ocs-consumer-cr-admins.ClusterRole.yml

```diff
@@ -71,4 +71,20 @@ rules:
       - watch
       - patch
       - update
-      - delete
\ No newline at end of file
+      - delete
+  - apiGroups:
+    - monitoring.coreos.com
+    resources:
+    - alertmanagers
+    - alertmanagerconfigs
+    - prometheuses
+    - prometheusrules
+    - servicemonitors
+    - podmonitors
+    verbs:
+      - get
+      - list
+      - watch
+      - patch
+      - update
+      - delete
```

Comment:
This is too permissive for a clusterrole for these resources.

Reply:
This ClusterRole bound via SubjectPermission to the openshift-storage NS only for backplane-odf.

Reply:
this is limited to the Addon namespace:

https://github.com/openshift/managed-cluster-config/blob/master/deploy/backplane/mtsre/ocs-consumer/10-mtsre-ocs-consumer-admins.SubjectPermission.yml#L11-L13

still an issue?